site/class-3-clean/mymphy/index.html

```diff
@@ -120,20 +120,23 @@
   doc = doc.replace('#', '').trim();
   doc = doc.replace(/ +/g, '')
   
-  var parseTime = d3.timeParse("%Y%m%d%H")
+  var parseTime = d3.timeParse('%Y%m%d%H')
+
   var data = d3.csvParseRows(doc, map)
   function map(d) {
+    if (d[7] === '') return;

```

Comment:
Have you tested if this works? 🤔

Reply:
Ja ik heb het getest en het werkt. Is dit niet de juiste oplossing ?

Reply:
Je was de eerste die het op deze manier deed!